src/core/vectortile/qgsvectortileloader.cpp

```diff
@@ -130,6 +130,13 @@ void QgsVectorTileLoader::tileReplyFinished()
   }
   else
   {
+    if ( reply->error() == QNetworkReply::ContentAccessDenied )
+    {
+      mError = tr( "Access denied" );
+      if ( !reply->data().isEmpty() )
+        mError.append( QStringLiteral( ": " ) + reply->data() );
```

Comment:
```suggestion
      mError = !reply->data().isEmpty() ? tr( "Access denied: %1" ).arg( reply->data() ) : tr( "Access denied" );
```

See above note about ":" requiring translation.